packages/kit/src/core/prerender/prerender.js

```diff
@@ -155,6 +163,12 @@ export async function prerender() {
 	const seen = new Set();
 	const written = new Set();
 
+	/** @type {Map<string, Set<string>>} */
+	const expected_deeplinks = new Map();
+
+	/** @type {Map<string, string[]>} */
+	const actual_deeplinks = new Map();
```

Comment:
I was confused for a moment because I understand a regular link to not just the home page by "deeplink", I think something like "hashlink" would be better.

Reply:
agree — fixed